beam/core/src/main/java/cz/o2/proxima/beam/direct/io/CommitLogRead.java

```diff
@@ -166,6 +192,7 @@ public boolean onError(Throwable error) {
 
         @Override
         public boolean onNext(StreamElement ingest, OnNextContext context) {
+          context.nack();
```

Comment:
why is this needed?

Reply:
Because otherwise we would hold the consumed element until timeout (or forever in case of `ListCommitLog`, which doesn't timeout elements).

Reply:
👍 makes sense, thanks